app/APIStart.py

```diff
@@ -8,148 +8,206 @@
 from flask import Flask
 from flask import request
 from flask import jsonify
-import json
-from DBhandler import DatabaseHandler as db
-from Items import Item as item
-from User import User
+from app.User import User, UserBase
+from app.Item import Item, ItemBase
+from sqlalchemy import create_engine
+from flask_sqlalchemy import SQLAlchemy
 
 app = Flask(__name__)
```

Comment:
this should not be here, look at how to organize flask app
https://exploreflask.com/en/latest/organizing.html

Reply:
Ok looking into it